### PR TITLE
docs: polish ProdMon examples

### PR DESCRIPTION
**Pull request by staceysv**
ProdMon overview example
* add imports for dependencies: tensorlfow, gradio, weave
* add wanbd login and make sure to set entity != Tim
* wording polish: step index, typos, clarification
* TODO: currently erroring on gradio authentication? will double-check

### CODE REVIEW

**Comment by staceysv** [comment]
cleaned up examples! there's a new error in the last cell of the ProductionMonitoring one that looks like gradio authentication/setup/maybe a changed api?
@tssweeney any chance you could take a look? also possible i'm not installing gradio correctly

File: examples/experimental/ProductionMonitoring/ProductionMonitoringConceptualOverview.ipynb [hunk 1]
```diff
@@ -6,29 +6,63 @@
...
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "2427bb8e",
+   "id": "424dc619",
    "metadata": {},
    "outputs": [],
    "source": [
-    "%autoreload 2"
+    "WB_ENTITY = \"stacey\"\n",
```
Comment (scottire) [inline examples/experimental/ProductionMonitoring/ProductionMonitoringConceptualOverview.ipynb:63]: I think we remove your entity here.

Comment (scottire) [reply examples/experimental/ProductionMonitoring/ProductionMonitoringConceptualOverview.ipynb:63]: other than this, I think it's ready to merge

Comment (staceysv) [reply examples/experimental/ProductionMonitoring/ProductionMonitoringConceptualOverview.ipynb:63]: oops thank you was iterating many times and forgot!

**Comment by staceysv** [comment]
@scottire verified ProdMon gradio works with a clean env, this should be good to go!

**Review by scottire — APPROVED** [review]

**Pull request merged**